WMFModel/WMFArticle+CoreDataProperties.h

```diff
@@ -0,0 +1,29 @@
+#import "WMFArticle+CoreDataClass.h"
+
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface WMFArticle (CoreDataProperties)
+
++ (NSFetchRequest<WMFArticle *> *)fetchRequest;
+
+@property (nonatomic) BOOL isBlocked;
+@property (nullable, nonatomic, copy) NSString *key;
+@property (nullable, nonatomic, copy) NSDate *viewedDate;
+@property (nullable, nonatomic, copy) NSDate *viewedCalendarDate;
```

Comment:
Not clear what this property is from the name. Looks like it is the date without time? Is that right? If so, maybe call it something like that viewedDateWithoutTime?